ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__lp-interop-ocp4.12.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  upi-installer:
```

Comment:
Is upi-installer needed here or can this run without it?

Reply:
I don't think so, just left over from the previous iteration. Removed in [3fd91bd](https://github.com/openshift/release/pull/36181/commits/3fd91bd72531fc6e504ff9eb26657e0278dea6d1)